modules/nodes-nodes-resources-configuring-auto.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-resources-configuring.adoc
+
+[id="nodes-nodes-resources-configuring-auto_{context}"]
+= Automatically allocating resources for nodes
+
+{product-title} can automatically determine the optimal sizing value of the `system-reserved` setting when a node starts.  
+
+When automatic resource allocation is enabled, a script on each node calculates the optimal values for the respective reserved resources based on the installed capacity for CPU and memory on the node. The script take into account that increased capacity requires a corresponding increase in the reserved resources. 
```

Comment:
@harche Excellent point. I removed this phrase _When automatic resource allocation is enabled_. It does sound like an official feature name. I didn't use the _Automatic Node Sizing_ wording because the rest of the topic talks about allocating resources. if you think we should go with node sizing, let me know. Maybe we can ask product management or product experience what they think. 
https://github.com/openshift/openshift-docs/pull/31543/commits/09f5a901e820c5ddc63fc8ee12693206798fac0f#diff-e77ebc7f0bfc0bf653d9998ef2e517ddd64c99798c7548473b175fbc1e9ab707R10